src/components/editMeasure/testCases/components/testCaseLanding/common/TestCaseTable/TestCaseTable.tsx

```diff
@@ -418,7 +418,7 @@ const TestCaseTable = (props: TestCaseTableProps) => {
           open={deleteDialogModalOpen}
           onContinue={() => {
             if (featureFlags?.TestCaseListActionCenter) {
-              deleteTestCase(selectedTestCases[0].id);
+              deleteTestCase();
               setDeleteDialogModalOpen(false);
             } else {
               deleteTestCase(selectedTestCase.id);
```

Comment:
please try deleting a single test case by setting `TestCaseListActionCenter` to false. I think it breaks. the reason is  `setDeleteDialogModalOpen(false);` is scopped to `if` block. I think it should be out side of `if else`

Reply:
Oh yep. Fixed